Integrations/integration-CheckpointFirewall.yml

```diff
@@ -145,12 +151,62 @@ script:
             }
             return ret;
         }
-    };
+    }
 
-    var sendRequest = function(cmdURL, body, cmdHeaders) {
-        var headers = cmdHeaders? cmdHeaders : {'content-type': ['application/json'], 'X-chkp-sid' : [sid]};
+
+    function prettify_show_hosts(hosts){
+        var pretty_hosts = [];
+        for (i = 0; i < hosts.length; i++) {
+            pretty_hosts[i] = {
+                'UID':  hosts[i].uid,
+                'Name':  hosts[i].name,
+                'IPV4':  hosts[i]['ipv4-address'],
+                'Domain': 'Name: ' + hosts[i].domain.name + ', UID: ' + hosts[i].domain.uid + ', Type: ' + hosts[i].domain['domain-type'],
```

Comment:
are we sure that all values exist? `hosts[i].domain.name`/` hosts[i].domain.uid`/`hosts[i].domain['domain-type']` and so on?
maybe we should test before?